### PR TITLE
Feature#23/add the review comments and score after the reservation is finished

### LINKED ISSUE #23: Add the review comments and score after the reservation is finished
# REQUIREMENTS
- After the reservation has been finished the guest can add a comment and score the host service
- Global score should be update after the review
- Host must not be available to make comments or score his own house

# DESIGN
Not neccesary

### PR DESCRIPTION
**Pull request by Frcomparan**
# Description
What did you implemented/Why did you implemented this:
What?
 - Added function to left a reviews when a reservation is finished
 - Added function to recalculate a home's score after review creation
 - Added a schedule to update the status of a home at the beginning of the day
 - Added some styles to reviews views

Why?
  - To allow the guest leave a comment about the service
  - To always have the home score updated
  - To update the status of reservations automatically

  ## Screenshots 
image

### CODE REVIEW

File: app/assets/stylesheets/abstracts/_classes.scss [hunk 1]
```diff
@@ -155,6 +159,18 @@
 .actions {
   @include center-grid;
   margin-top: 1rem;
+
+  input {
+    border-radius: 8px;
+    padding: 8px;
+    border: none;
+    box-shadow: $low-shadow;
```
Comment (dannharo) [inline app/assets/stylesheets/abstracts/_classes.scss]: sort this

Comment (dannharo) [reply app/assets/stylesheets/abstracts/_classes.scss]: also [here](https://github.com/BC-MAY-21-ROR/proyecto-final-pf-f22-t4/pull/38/files#diff-a73ffa8bf665f352b7e714655119ac5eeacd69b3dcafc618da05eeda2a1a70d1R172)

**Review by JuanequeX — DISMISSED** [review]
LGTM

**Review by MrCano369x — APPROVED** [review]
lgtb

File: app/assets/stylesheets/abstracts/_classes.scss (hunk 1 above)
Comment (Frcomparan) [reply app/assets/stylesheets/abstracts/_classes.scss]: Sorted

**Review by HeikelV — APPROVED** [review]
LGTM

**Pull request merged**